src/rotator_library/anthropic_compat/streaming_fast.py

```
@@ -0,0 +1,460 @@
+                        transaction_logger, request_id, original_model,
+                        accumulated_thinking, accumulated_text,
+                        tool_calls_by_index, input_tokens, output_tokens,
+                        cached_tokens, stop_reason_final
+                    )
+                break
+
+            # Parse chunk (fast path with orjson)
+            try:
+                chunk = json_loads(data_content)
+            except Exception:
+                continue
```

Ruff (0.15.4)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)